oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/status/ServerStatusService.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package org.apache.skywalking.oap.server.core.status;
+
+import lombok.RequiredArgsConstructor;
+import org.apache.skywalking.oap.server.library.module.ModuleManager;
+import org.apache.skywalking.oap.server.library.module.Service;
+import org.apache.skywalking.oap.server.telemetry.TelemetryModule;
+import org.apache.skywalking.oap.server.telemetry.api.MetricsCreator;
+import org.apache.skywalking.oap.server.telemetry.api.MetricsTag;
+
+/**
+ * The server status service provides the indicators for the current server status.
+ * Notice, this should not be treated as a kind of health checker or self telemetry.
+ * For more, this helps modules to be aware of current OAP server status.
+ *
+ * @since 9.4.0
+ */
+@RequiredArgsConstructor
+public class ServerStatusService implements Service {
+    private final ModuleManager manager;
+    private BootingStatus bootingStatus = new BootingStatus();
+
+    public void bootedNow(long uptime) {
+        bootingStatus.setBooted(true);
+        bootingStatus.setUptime(uptime);
+        manager.find(TelemetryModule.NAME)
+               .provider()
+               .getService(MetricsCreator.class)
+               .createGauge("uptime", "oap server start up time", MetricsTag.EMPTY_KEY, MetricsTag.EMPTY_VALUE)
+               // Set uptime to second
+               .setValue(uptime / 1000d);
```

Comment:
I moved the logic here to keep consistent. @kezhenxu94

Reply:
The next step we should need a registration and notification mechanism in this service for other modules, which could sense cluster module update events or latest status.

Reply:
This doesn't make sense to me. You are essentially proposing to replicate the health checker module in the core module again. What's the boundary to do or not to do this kind of stuff in your mind? You said you don't want to replace the health checker module but it can not persuade me and we have no reason to stop future contributors from doing similar things. The new server status service and the old health checker are the same to me except for the fact that they are written by different authors.

Reply:
In next PR, I need to read boot time, I don't think this makes sense to read data from health checker. 

Thr boundaries are health check is for exposing true false status and error message for out of the process. The core service is for modules in the OAP to interact or read status.

Reply:
At least, when @hanahmily added health checker, it only cares the OAP is running or not, rather than inside details. 
For example, we are not going to expose cluster information to checker, or out of process. Right?

From data point, uptime is already  both in health check and telemetry, two components. .

Reply:
> You are essentially proposing to replicate the health checker module in the core module again

I am not, I am not sure why you think this way. I just add the boot status, because I just need it for the next step. But in https://github.com/apache/skywalking/issues/10051, more status relative mechanism is going to be added.